docs/api/scripts_refactor_lint_report_pkg_plugins.md

```
@@ -0,0 +1,130 @@
+**Returns:**
+int: The exit code from the Black command.
+
+#### `parse`
+Parse the output report from Black and update the destination dictionary.
+**Parameters:**
+dst (Dict[str, Dict[str, Any]]): Destination dictionary to update with formatting needs.
+
+
+## `scripts\refactor\lint_report_pkg\plugins\flake8`
+
+**🧠 Docstring Summary**
```

markdownlint-cli2 (0.17.2)

Emphasis used instead of a heading
null

(MD036, no-emphasis-as-heading)